Fixed space diference between texts and logos https://github.com/Israel-Laguan/fast-shopping-backend

Dajava02 opened the pull request:
## Explanation

- I put the same space between each text and logo
- I also activate the link to linkedin

## Screenshots:

before:

![espacio original](https://user-images.githubusercontent.com/105750239/169600827-b33752a0-b0f5-4fdd-b834-ceaacb7f722e.PNG)

After:

![espacio con el 3er cambio](https://user-images.githubusercontent.com/105750239/169600879-563b3330-bf16-4f6f-aef6-d564096b3abd.PNG)

Dajava02 committed:
Fixed space diference between texts and logos

Explanation

I put the same space between each text and logo
I also activate the link to linkedin

Screenshots:

before:

After:

Israel-Laguan commented on README.md:
```diff
@@ -115,14 +115,14 @@ For more information about the code please check the [code overview](docs/CODE_O
             </a>
             <a href="https://www.linkedin.com/in/israellaguan/" target="_blank" rel="author">
-                <img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="My Linkedin" height="45px">
+                <img src="https://img.icons8.com/color/48/000000/linkedin.png" style="border-radius: 10%" alt="My Linkedin" height="45px">
```
`style="border-radius: 10%"` seems like is not working on GitHub

Dajava02 committed:
Change in spaces and content

Explanation

Remove style="border-radius: 10%"

Change the separation of texts and logos

Change the alt text in the mail image from "My GitHub" to "My Email"

Fill in the underline separation in "Email me to israellaguan@gmail.com"
